Makefile

```diff
@@ -0,0 +1,8 @@
+# live-check observes all .go files outside of the vendor tree and whenever any
+# of them changes runs "go test ./..." which tests the whole tree. This is
+# handy to keep running in a separate window (on a 2nd monitor perhaps) while
+# editing code as it allows one to not have to switch to another application to
+# see the general status of unit tests.
+.PHONY: live-check
+live-check:
+	find -name "*.go" \! -path "./vendor/*" | entr -c go test ./...
```

Comment:
Quite sure there's a community tool for this.

Reply:
Do you know the name of that tool?

Reply:
There's https://github.com/go-godo/godo and it doesn't need a makefile (makes it less controversial?). OTOH it should be fairly simple to write something using `fsnotify`, with a configurable settle time and perhaps exploiting the lower level Go toolchain information.